gengokit/template/template.go

```diff
@@ -279,7 +279,7 @@ func nameServiceHandlersServerServer_handlerGotemplate() (*asset, error) {
 	return a, nil
 }
 
-var _nameServiceMiddlewaresEndpointsGotemplate = []byte("\x1f\x8b\x08\x00\x00\x09\x6e\x88\x00\xff\x04\xc0\x41\x0a\xc4\x20\x0c\x05\xd0\xfd\x9c\x22\x6b\x61\x26\x87\x99\x13\x48\xf3\x1b\xa4\x6a\xc4\xa4\x74\x21\xde\xbd\x6f\x2d\x4e\xf4\x07\x48\xed\x1b\xf3\x76\x67\x45\x57\xbb\x4a\x70\x2b\x22\x15\x4f\x9e\x70\x0e\xb4\x51\x73\xc0\x19\x5d\x86\x95\x1e\xfe\x53\xa3\xd3\x26\x1d\x26\xa0\xc4\x7b\x7f\xde\x00\x00\x00\xff\xff\x80\x45\x66\x4f\x4c\x00\x00\x00")
+var _nameServiceMiddlewaresEndpointsGotemplate = []byte("\x1f\x8b\x08\x00\x00\x09\x6e\x88\x00\xff\x0c\xc6\xc1\x0d\xc3\x20\x0c\x05\xd0\x7b\xa7\xf0\x19\xa9\xf5\x30\x9d\x00\xd5\xbf\x16\x0a\x60\x84\x1d\xe5\x80\xd8\x3d\xb9\xbd\xb5\x38\xd1\x17\x20\xb5\x77\xcc\xd3\x9d\x15\x5d\xed\x28\xc1\xad\x88\x54\x5c\x79\xc2\x39\xd0\x46\xcd\xf1\x08\x5d\x86\x95\x1e\xfe\x51\xa3\xbf\x4d\xfa\x99\x80\x12\xef\xfd\xba\x03\x00\x00\xff\xff\x80\x45\x66\x4f\x4c\x00\x00\x00")
```

Comment:
I suspect this is the result of an outdated `go-bindata`. Updating, re-running and pushing now to confirm.